src/elm/Page/Community.elm

```diff
@@ -9,23 +9,18 @@ module Page.Community exposing
     , view
     )
 
-import Api
```

Comment:
1k lines removal from our biggest and messier file on the project:
![so_good](https://user-images.githubusercontent.com/1082127/107817980-a3a3c980-6d55-11eb-980d-35b41fc63568.jpg)